CONTRIBUTION.md

```diff
@@ -0,0 +1,184 @@
+# Contributing to Antenna
+
+First off, thanks for contributing!
+
+This is a set of guidelines for contributing to Antenna, which is hosted by the [Eclipse Foundation](https://github.com/eclipse) on Github.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [Eclipse Contributor Agreement](#eclipse-contributor-agreement)
+  * [Contact](#contact)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [Java Styleguide](#java-styleguide)
+  * [Documentation Styleguide](#documentation-styleguide)
+
+## Code of Conduct
+
+This project is hosted by the Eclipse foundation and hence adheres to the 
+[Eclipse Code of Conduct](https://wiki.eclipse.org/Eclipse_Code_of_Conduct) that is based on the
+[Contributor Covenant](https://www.contributor-covenant.org/).
+
+## What should I know before I get started?
+
+### Eclipse Contributor Agreement
+
+Before your contribution can be accepted by the project team contributors must
+electronically sign the Eclipse Contributor Agreement (ECA).
+
+* http://www.eclipse.org/legal/ECA.php
+
+Commits that are provided by non-committers must have a Signed-off-by field in
+the footer indicating that the author is aware of the terms by which the
+contribution has been provided to the project.
+To achieve this, sign your commits by adding `-s` to your `git commit` command. 
+The non-committer must additionally have an Eclipse Foundation account and 
+must have a signed Eclipse Contributor Agreement (ECA) on file.
+Your git signature must match your Eclipse Foundation account information to be accepted.
```

Comment:
Nit: When not referring to the command, but the tool in general, "Git" should be capitalized.